Makefile.machine.include

```diff
@@ -31,6 +31,7 @@ BSG_MACHINE_VCACHE_SET                = 64
 BSG_MACHINE_VCACHE_WAY                = 8
 BSG_MACHINE_VCACHE_BLOCK_SIZE_WORDS   = 16
 BSG_MACHINE_VCACHE_STRIPE_SIZE_WORDS  = $(BSG_MACHINE_VCACHE_BLOCK_SIZE_WORDS)
+BSG_MACHINE_VCACHE_DMA_DATA_WIDTH     = 32
 BSG_MACHINE_VCACHE_MISS_FIFO_ELS      = 32
 BSG_MACHINE_DRAM_INCLUDED             = 1
```

Comment:
This is perhaps a little pedantic, but it'd be nice if this were grouped with CL_MANYCORE_DRAM_CHANNELS

Reply:
Right now this lines up with where the makefiles in bsg_manycore put it. I can move where CL_MANYCORE_DRAM_CHANNELS is defined but that's something specific to our testbench.

Reply:
Best to stick with what Manycore has. Ignore this